docs/faq.rst

```diff
@@ -13,7 +13,7 @@ Yes, the `deduplication`_ technique used by
 Also, we have optional simple sparse file support for extract.
 
 If you use non-snapshotting backup tools like Borg to back up virtual machines,
-then these should be turned off for doing so. Backing up live VMs this way can (and will)
+then the VMs should be turned off before doing so. Backing up live VMs can (and will)
```

Comment:
As a native (US) English speaker, I found the "turned off for doing so" language difficult to parse.  I eventually assumed "for" was intended to be "before" and then it made sense.

"turned off for the duration of the backup" would be clearer and would get across your point that it must stay off.

BTW: As a long time Attic user, I just discovered Borg yesterday.  Thank you for your efforts to revitalize development.